types/react/index.d.ts

```diff
@@ -349,13 +349,6 @@ declare namespace React {
 
         /** A fallback react tree to show when a Suspense child (like React.lazy) suspends */
         fallback: NonNullable<ReactNode>|null;
-
-        // I tried looking at the code but I have no idea what it does.
-        // https://github.com/facebook/react/issues/13206#issuecomment-432489986
-        /**
-         * Not implemented yet, requires unstable_ConcurrentMode
-         */
-        // maxDuration?: number;
```

Comment:
Both this and unstable_ConcurrentMode were removed. unstable_ConcurrentMode was never added to the types anyway.